d/dotnet7/prereq_install

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+
+apt install -y dpkg-dev
+
+dpkg --add-architecture ppc64el
+sudo apt-get update
```

Comment:
probably don't need sudo here as the script is run as a root user? it can fail if run in a fresh container as sudo is not installed there by default